include/surface_perception/shape_extraction.h

```diff
@@ -33,6 +33,36 @@ bool FitBox(const pcl::PointCloud<pcl::PointXYZRGB>::Ptr& input,
             const pcl::PointIndicesPtr& indices,
             const pcl::ModelCoefficients::Ptr& model, geometry_msgs::Pose* pose,
             geometry_msgs::Vector3* dimensions);
+
+/// \brief Returns a standardized orientation for a box.
+///
+/// The standardized box orientation is defined as the following:
+///  1. If x dimension of the box must be smaller than or equal to y dimension.
```

Comment:
Remove the "If." Just say,
1. The x dimension of the box will be smaller than the y dimension of the box.
2. The x-axis of the box will point in the positive x direction.
3. The z-axis will point in the same direction as given, and the y-axis will be set according to the right hand rule.

Reply:
Completed